src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,23 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
+    @NotNull
+    @Size(min = 6)
     private String password;
```

Comment:
Put all checks you need in @Password annotation check. Do not use @NotNull and @Size with `password` or `repeatPassword`
![image](https://user-images.githubusercontent.com/20326858/233617800-fe9fcc19-7282-478e-9251-45119913b71e.png)